<launch>
  <arg name="drone_id" default = "0"/>
  <arg name="odometry_topic" default = "lidar_slam/odom"/>
  <node pkg="guide_manager" name="guide_manager" type="guide_plan.py" output="screen" ns="/drone_$(arg drone_id)" respawn="true">
    <param name="drone_id" value="$(arg drone_id)" />
    <param name="ugv_num" value="7" />
    <param name="uav_num" value="1" />
    <param name="replan_period" value="10.0" />
    <param name="v_max" value="3.0" />
    <param name="a_max" value="1.0" />
    <param name="time_reslolution" value="100.0" />

    <remap from="/self_odom" to="/drone_$(arg drone_id)/$(arg odometry_topic)"/>
    <remap from="/guide_waypoint" to="/drone_$(arg drone_id)/guide_waypoint"/>
    <remap from="/blind_info" to="/drone_$(arg drone_id)/broadcast/blind_info"/>
  </node>
</launch> 
